falco-exporter/values.yaml

```diff
@@ -95,4 +98,8 @@ grafanaDashboard:
 
 scc:
   # true here enabled creation of Security Context Constraints in Openshift
-  create: true
\ No newline at end of file
+  create: true
+
+# Enable Mutual TLS for HTTP metrics server
+mTLS:
+  enabled: false
```

Comment:
Just a thought about the naming.
Since mutual TLS is also a setting for the gRPC, I'm a bit concerned it might be confusing.
What about a more explicit name?
Otherwise, we may put `mTLS` under the `service` configuration node since it actually refers to the exposed service.
WDYT?

Reply:
You're right, I'll change it

Reply:
You're right, I'll change it

Reply:
Done, and NOTES.txt template has been updated too